spec/validates_zipcode_spec.rb

```diff
@@ -143,11 +143,15 @@
       zipcode_should_be_valid(record)
       record = build_record('LT-00110', 'LT')
       zipcode_should_be_valid(record)
+      record = build_record('0110', 'LT')
+      zipcode_should_be_valid(record)
+      record = build_record('00110', 'LT')
+      zipcode_should_be_valid(record)
```

Comment:
Could you please add test cases for the `LT-` format, and invalid with `LT` without dash, please?

Reply:
add more tests in latest commit. let me know if you want anymore.